<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPanel</class>
 <widget class="QWidget" name="ConfigPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Difficulté</string>
      </property>
      <widget class="QRadioButton" name="easyMode">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>359</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Facile</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>easyMode</zorder>
      <zorder>mediumMode</zorder>
      <zorder>hardMode</zorder>
      <zorder>easyMode</zorder>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="hardMode">
      <property name="text">
       <string>Difficile</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="mediumMode">
      <property name="text">
       <string>Moyen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Type de reprise</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="activateLoop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Boucler sur les parties</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="continueType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <zorder>cancelButton</zorder>
  <zorder>okButton</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>continueType</zorder>
  <zorder>label_2</zorder>
  <zorder>label_2</zorder>
  <zorder>activateLoop</zorder>
  <zorder>continueType</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>pressed()</signal>
   <receiver>ConfigPanel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>pressed()</signal>
   <receiver>ConfigPanel</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveData()</slot>
 </slots>
</ui>
